public/content/lessons/2017/chain-rule-and-product-rule/index.mdx

```diff
@@ -7,4 +7,355 @@ video: YG15m2VwSjA
+
+<Accordion title="Explanation">
+
+One way to verify that this is the case is to visualize a constant multiplied by a function as the area of the rectangle with width equal to the function and height equal to the constant. 
+
+<Figure
+  image="figures/constant-multiplication-area.svg" 
+/>
+
+Then, introducing a small nudge $dx$ to the input of the function, you can measure the change in the area highlighted in yellow below.
+
+<Figure
+  image="figures/constant-multiplication-area-df.svg" 
+/>
+
+In the case of $g(x) = \sin(x)$ and $c = 2$ we know that $d(g(x)) = \cos(x) \cdot dx$. This gives us a change in area of $df = 2 \cdot \cos(x) \cdot dx$. Solving for $\frac{df}{dx}$ we get that the derivative is $2\cos(x)$.
+
+</Accordion>
+
+## Chain rule
+
+Aside from addition and multiplication, the other common way to combine functions that comes up all the time is function composition. For example, let’s say we take the function $x^2$, and shove it inside $\sin(x)$ to get a new function, $\sin(x^2)$. Or, in other words, the output of the function $x^2$ gets fed as input to sine function.
+
+What’s the derivative of this new function?
+
+Here I’ll choose yet another way to visualize things, just to emphasize that in creative math, we have lots of options. I’ll put up three number lines. The top one will hold the value of $x$, the second one will represent the value of $x^2$, and the third line will hold the value of $\sin(x^2)$. 
+
+<Figure
+  image="figures/chain-rule-x-1-2.svg" 
+/>
+
+That is, the function $x^2$ gets you from line $1$ to line $2$, and the function sine gets you from line $2$ to line $3$.
+
+<Question
+  question="What is the value of the composed function given the input $x=2$. To get a hang of this visualization technique, go from the first line to the second line to the third line."
+  choice1="$\sin(2) \approx 0.909$"
+  choice2="$\sin(4) \approx -0.757$"
+  answer={2}
+>
+
+<Figure
+  image="figures/chain-rule-x-2.svg" 
+/>
+
+Using the number line visual, start on $x=2$ and move to the next line using the function $g(x) = x^2$. Plugging in $x=2$ into the function we get $(2)^2 = 4$ on the next numberline which we then feed into the outer function $h(x) = \sin(x)$. 
+
+</Question>
+
+As I shift that value of $x$, maybe up to the value $3$, then the value on the second shifts to whatever $x^2$ is, in this case $9$. And that bottom value, being the $\sin(x^2)$, will go over to whatever the $\sin(9)$ is.
+
+<Figure
+  image="figures/chain-rule-x-3.svg" 
+/>
+
+So for the derivative, let’s again think of nudging that $x$-value by some little $dx$, and I always think it’s helpful to think of $x$ starting as some actual number, maybe $1.5$. And $dx$ as some small number that I imagine approaching zero, maybe $0.1$.
```

Comment:
```suggestion
So for the derivative, let’s again think of nudging that $x$-value by some little $dx$. I find it helpful to imagine $x$ starting out as some actual number, maybe $1.5$, and $dx$ as some small number approaching zero, like $0.1$.
```

Again, the transcript from the video feels weird when written. Changing the words a bit makes it easier to read, at least to me.